compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2265,6 +2267,8 @@ pub struct TargetOptions {
     pub is_like_wasm: bool,
     /// Whether a target toolchain is like Android, implying a Linux kernel and a Bionic libc
     pub is_like_android: bool,
+    /// Whether a target toolchain is like VEXos.
```

Comment:
"VEXos" is not well-known or [googleable](https://www.google.com/search?q=vexos).

```suggestion
    /// Whether a target toolchain is like VEXos (VEX Robotics V5 Brain's OS).
```

Reply:
Or maybe just mention the target its for there?